content/docs/3_reference/3_panel/3_fields/0_blocks/reference-article.txt

```diff
@@ -233,6 +233,11 @@ To edit a custom block type, editors can either double-tap on the block or click
 
 (image: editing.png)
 
+<since v="3.6.0">
+#### Drawer-less blocks
+If you don't define any `fields`, the block won't have a drawer. This can be useful for blocks that don't feature any configuration (e.g. the default `line` clock).

```

Comment:
```suggestion
If you don't define any `fields`, the block won't have a drawer. This can be useful for blocks that don't feature any configuration (e.g. the default `line` block).

```